standardtracer/span.go

```diff
@@ -45,14 +56,14 @@ func (s *spanImpl) LogEventWithPayload(event string, payload interface{}) {
 }
 
 func (s *spanImpl) Log(ld opentracing.LogData) {
-	s.lock.Lock()
-	defer s.lock.Unlock()
+	s.Lock()
+	defer s.Unlock()
 
 	if ld.Timestamp.IsZero() {
 		ld.Timestamp = time.Now()
 	}
 
-	s.raw.Logs = append(s.raw.Logs, &ld)
+	s.raw.Logs = append(s.raw.Logs, ld)
```

Comment:
for large LogData instances I'm surprised that this is a perf improvement! but that's what benchmarks are for, I guess.